modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java

```diff
@@ -822,8 +822,7 @@ private Hits _getHits(SearchContainer<RepositoryEntry> searchContainer)
 	private String _getPortletPreference(String name, String defaultValue) {
 		if (_themeDisplay.isSignedIn()) {
 			try (SafeCloseable safeCloseable =
-					CTCollectionThreadLocal.
-						setProductionModeWithSafeCloseable()) {
+					CTCollectionThreadLocal.setSafeCloseable()) {
```

Comment:
Why can't we use CTCollectionThreadLocal.setCTCollectionIdWithSafeCloseable(CTCollectionThreadLocal.getCTCollectionId()) instead of adding a new method?  This will force a version bump in portal-kernel for no reason.